.gitmodules

```diff
@@ -46,7 +46,7 @@
 [submodule "src/sonic-frr/frr"]
 	path = src/sonic-frr/frr
 	url = https://github.com/sonic-net/sonic-frr.git
-	branch = frr/8.5
+	branch = frr-10.0.1
```

Comment:
any particular reason the convention for branch name has changed?  Make life easy for future people in figuring out how to go forwards/backwards to older/newer versions of sonic as much as is possible.

Reply:
The initial branch (frr/10.0.1) was created by the FRR convenor. But the branch did not point to the correct FRR10.0.1 commit-id. Later, the branch could not be deleted. Now, they have created a new branch frr-10.0.1, which points to the release commit of frr10.0.1.